docs/pages/06_expression.adoc

```diff
@@ -571,9 +572,10 @@ The percent sign represents zero, one, or multiple numbers or characters. The un
 [options="header",grid="none",cols="1a"]
 |======================================================================
 |`[purple]#**rpad(size = **#<size> &#x5B;[purple]##**, padstring = **##<padstring>][purple]##**)**## : string`
-|Fills up a string on the right to the specified <size> with <padstring>. If the length of <padstring> is less than the remaining length, the <padstring> is repeated until it is not filling up. If the length of <padstring> is longer than the remaining length, it will be truncated on the left. If <padstring> is not defined, one space character (`" "`) is used by default.
+|Fills up a string on the right to the specified <size> with <padstring>. If the string's length is less then than the specified <size> the string is truncated from the right to the specified <size>. If the length of <padstring> is less than the remaining length, the <padstring> is repeated until it is not filling up. If the length of <padstring> is longer than the remaining length, it will be truncated on the left. If <padstring> is not defined, one space character (`" "`) is used by default.
```

Comment:
I think, it isn't true.
"If the string's length is less then than the specified `<size>` the string is truncated from the right to the specified `<size>`."

I think the correct santence is the following.
"If the string's length is more then than the specified `<size>` the string is truncated from the right to the specified `<size>`."

Reply:
https://github.com/BlackBeltTechnology/judo-meta-jsl/pull/123/commits/941ae2b9c911902e2d0c5420b6f0ac889d3763d0